homework-selects/sql_questions.sql

```diff
@@ -3,55 +3,460 @@
 SELECT first_name, last_name, salary, job_title
 FROM employees
          LEFT JOIN jobs USING (job_id);
+
 -- 2. the first and last name, department, city, and state province for each employee.
+SELECT first_name, last_name, department_name, city, state_province
+FROM employees
+    LEFT JOIN departments USING (department_id)
+    LEFT JOIN locations USING (location_id);
+
 -- 3. the first name, last name, department number and department name, for all employees for departments 80 or 40.
+SELECT first_name, last_name, department_name, department_id
+FROM employees
+    LEFT JOIN departments USING (department_id)
+WHERE department_id = 80 or department_id = 40;
+
 -- 4. those employees who contain a letter z to their first name and also display their last name, department, city, and state province.
+SELECT first_name, last_name, department_name, city, state_province
+FROM employees
+    LEFT JOIN departments USING (department_id)
+    LEFT JOIN locations USING (location_id)
+WHERE first_name LIKE '%z%'
```

Comment:
Can we add an additional check, what if the employee's firstName starts with Z?

Reply:
Done